<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="confirmOrderTable">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>110</y>
     <width>721</width>
     <height>311</height>
    </rect>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <property name="columnCount">
    <number>4</number>
   </property>
   <column>
    <property name="text">
     <string>Menu ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Item Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Price</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Quantity</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Confirm Order</string>
   </property>
  </widget>
  <widget class="QPushButton" name="placeOrderBtn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>520</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Place Order</string>
   </property>
  </widget>
  <widget class="QPushButton" name="incQtyBtn">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>450</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Increase Qty</string>
   </property>
  </widget>
  <widget class="QPushButton" name="decQtyBtn">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>450</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Decrease Qty</string>
   </property>
  </widget>
  <widget class="QPushButton" name="removeItemsBtn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>450</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove Selected Items</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
